Mesh_3/include/CGAL/Mesh_3/polylines_to_protect.h

```diff
@@ -785,7 +785,7 @@ polylines_to_protect(const CGAL::Image_3& cgal_image,
      word_type,
      null,
      CGAL::Identity<Image_word_type>(),
-     internal::Mesh_3::Returns_midpoint<K, Image_word_type>(),
+     Mesh_3::internal::Returns_midpoint<K, Image_word_type>(),
```

Comment:
I do not understand what happened! `CGAL::internal::Mesh_3` should be the right namespace, according to our coding guidelines.